feat(AP-3811): Switch between active combinations

joshua-barnard-evolv opened the pull request:
1. You can now see and switch between active combinations for any active project. The extension fetches the environment config to see the active combinations.
2. Projects are now sorted by confirmations, then name.
3. If you have selected a combination, that project appears at the top of the list.
4. A "clear selection" link appears if you select a different combination than the one allocated. You become excluded from all other projects until you click the link, which wipes the session storage item `evolv:previewCid`.
5. Added a `background.js` service worker as per the developer docs and updated the manifest to prevent CORS issues accessing the environment config.
6. Appended "(C)" to the control combinations.

![image](https://github.com/evolv-ai/chrome-extension/assets/78709720/ca984254-6a0d-4038-83e5-b4c1efcf056f)


https://github.com/evolv-ai/chrome-extension/assets/78709720/b454c4e8-4e6c-4360-8379-53f00a3f6d55

robertsevernsentient commented on popup.js:
```diff
@@ -166,6 +234,22 @@ const setAllocationsAndConfirmations = () => {
 };
 
+const getCombinationLabel = (candidates, ordinal) => {
+  const isControl = ordinal === candidates[0].ordinal;
```
This assumes the candidate that is firs tin the list is control. I'm not sure we can guarantee that. We do know it is the combo with the lowest ordinal

joshua-barnard-evolv replied:
I've added sort.

robertsevernsentient commented on manifest.json:
```diff
@@ -1,12 +1,20 @@
+    "https://participants-stg.evolv.ai/*"
   ],
-  "permissions": [],
+  "permissions": [
```
test removing these

joshua-barnard-evolv replied:
Looks like it doesn't need these anymore. They were required for a different implementation I tried.

joshua-barnard-evolv commented:
I've addressed all of the PR feedback and modified the layout and styling for how we show combinations and Control.

<img width="509" alt="image" src="https://github.com/evolv-ai/chrome-extension/assets/78709720/e74bf2c7-6ef6-40c3-a9b0-839677e69dd8">
